addons/rose/app/styles/rose/_typography.scss

```diff
@@ -12,60 +12,26 @@ html {
 }
 
 body {
+  // Used by Rose::Tabs and '/' in breadcrumbs
+  // and link list panel
   font-family: typography.$family-primary;
 
-  // This is the only time we'll explicitly use pixels.  All other sizes are
-  // expressed in rem units.
-  font-size: typography.$base-font-size * 1px;
+  // these 3 are used across the app
   line-height: typography.line-height(m);
   color: var(--stark);
   background-color: var(--subtle);
 }
 
-input,
-textarea,
-button {
-  font-family: typography.$family-primary;
-}
-
-pre,
-code {
-  font-family: typography.$family-monospace;
-}
-
-h1,
-.h1 {
-  @include type-mixins.type(h1, bold);
-
-  margin-top: sizing.rems(m);
-  margin-bottom: sizing.rems(xs);
-}
-
-h2,
-.h2 {
-  @include type-mixins.type(h2, bold);
-
+.header {
```

Comment:
I did not realize when I created the `header-spacing` (now removed) rule that we had rules for the different levels of headers. So I decided to merge the rules into one called `header`. Happy to change the naming or location if needed.

Reply:
I notice that we removed the header (`h1, .h1`, etc) styles but we have all these individual headers now with the `header` class attached to them all, is there a reason we shouldn't have kept them?

It seems to me that's the rule rather than the exception, are there more cases with headers that shouldn't have the spacing?

Reply:
Great question. The rules that were specific to the element, `h1` for example, were being ignored and the `margin: 0` from HDS was overriding the rule. When I added the class to a specific HDS component, the rule `.h1` was being applied. Since it was ignoring the element directly, I went the route of using a single class for any header that we want those styles applied to.

Reply:
Ahh I see that makes sense. But do we feel like we _should_ have applied it overall to these header elements? If we do, it seems like this might be closer to an HDS override to handle the CSS specificity issue (e.g. `h2[class*=hds-typography-display-]`. I just wanted to make sure using `.header` as a utility class is conscious choice we made in how we want to style and not just a technical one we solved to get around an issue.

Reply:
I would really recommend not to have to use `!important` since that can be very gnarly to deal with in the future, usually you can still get it done with the right CSS specificity

Reply:
Sorry the delay, I have been thinking over what direction we should go with this. Please let me know your thoughts before I start making the changes.

1. I think an override is a good idea. The majority of our headers need the top and bottom margin rule.
    - We can specify the override to only apply to header tags to be safe.
3. This brings me to the second point, we do have some headers that do not need the margin rule. We could have a special class, just for those specific cases. That way we only need to apply a special class if we don't want the margin added.
4. I would also agree that `!important` is not something we want unless absolutely needed.

Reply:
This has been updated. Please let me know your thougths.